src/controllers/likeController.js

```diff
@@ -0,0 +1,57 @@
+import * as ApplicationError from '../utils/errors/applicationsErrors';
+import { deleteOrCreate } from '../services/likeService';
+import { Facility, Like } from '../database/models';
+
+const likeOrUnlike = async (req, res) => {
+  try {
+    const { facility_id: facilityId } = req.params;
+    const { id: userId } = req.user;
+    const facilityExist = await Facility.findOne({ where: { id: facilityId } });
+    if (!facilityExist) {
+      return ApplicationError.notFoundError(
+        'This facility does not exist',
+        res
+      );
+    }
+    const { created } = await deleteOrCreate(Like, {facilityId, userId});  // deleteOrCreate(model, {where and item})
+    if (created) {
+      return res
+        .status(201)
+        .json({
+          status: 201,
+          message: `You have liked facility ${facilityId}`
+        });
+    }
+    return res
+      .status(200)
+      .json({
+        status: 200,
+        message: `You have unliked facility ${facilityId}`
+      });
+  } catch (error) {
+    return ApplicationError.internalServerError(error.message, res);
+  }
+};
+const likes = async (req, res) => {
+  try {
+    const { facility_id: facilityId } = req.params;
+    const facilityLikes = await Like.findAndCountAll({   // numbers of likes of an accomodation facility
+      where: { facilityId }
+    }).count;
+    if (!facilityLikes) {
+      return res
+        .status(200)
+        .json({ likes: 0, status: 200, message: `No likes for this facility` });
+    }
+    return res
+      .status(200)
+      .json({
```

Comment:
it is better to create utils for responses as we did for errors. refer to this https://github.com/atlp-rwanda/cod-be/pull/51/files#diff-2c43eb137af37d22e822c4fa683a4dffc0290ddbcffcc73add55b82864b5b6a0